SOURCE/rprocess_fitfrm_source.f90

```diff
@@ -21,21 +21,23 @@ subroutine rprocess_fitfrm_source(it,nt,tcenter)
 !- See Rosswog & Korobkin (2023) for details
 !--------------------------------------------------
   integer :: icol, i
-  real*8 :: helparr(gas_ncell),vexp(gas_ncell)
+  real*8 :: helparr(gas_ncell),vexp(gas_ncell),gas_matsrc_raw(gas_ncell), gas_matsrc_dyn(gas_ncell), gas_matsrc_wnd(gas_ncell)
   real*8 :: therm_frac(gas_ncell)
   real*8, parameter :: M_ref = 0.05d0*pc_msun  !< reference mass when computing vexp
 
 !---------------------------------------------------
 !- Radioactive species partitioning (hardcoded for now!)
 !  See Wollaeger et al. (2018), Sec. 2.2
 !---------------------------------------------------
-  real*8, parameter :: A_alpha = 1.3d-11 ! [g cm^-3 s]
+  real*8, parameter :: A_alpha = 1.2d-11 ! [g cm^-3 s]
   real*8, parameter :: A_beta  = 1.3d-11 ! [g cm^-3 s]
   real*8, parameter :: A_ff    = 0.2d-11 ! [g cm^-3 s]
   real*8, parameter :: X_alpha = 0.05d0
   real*8, parameter :: X_beta  = 0.20d0
   real*8, parameter :: X_gamma = 0.35d0
   real*8, parameter :: X_ff    = 0.00d0
+  real*8, parameter :: ye_w    = 0.37d0
```

Comment:
If this separate Ye approach is important, we'll have to add an extra column in input.str to set these per ejecta component.